src/controllers/courses/course-routes.ts

```diff
@@ -27,7 +26,7 @@ export default function (server: Hapi.Server, serverConfigs: IServerConfiguratio
                           3. All courses (includes courses from 1 and 2.)',
             response: {
                 schema: Joi.object({
-                    "facilitatingCourses": Joi.array().items(facilitatingCourseSchema),
+                    "facilitatingCourses": Joi.array().items(courseSchema),
```

Comment:
change the code such that both the backend doesn't send facilitatingCourses doesn't send it, and frontend doesn't need it. 

ab you aren't facilitator for a course anymore, ,we are going for a different way of permissions altogether

Reply:
Should I remove the facilitating courses concept also? and there should be only enrolled and available courses? @abhishekgupta92

Reply:
Yes, I agree